ads/onead.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {validateData} from '../3p/3p';
+
+/**
+ * @param {!Window} global
+ * @param {!Object} data
+ */
+export function onead(global, data) {
+  validateData(data, [], ['playmode', 'uid', 'pid', 'host']);
+  global.Guoshi = {
+    queryAd: {
+      amp: {},
+    },
+  };
+  global.ONEAD_AMP = {
+    playmode: data.playmode,
+    uid: data.uid,
+    pid: data.pid,
+    host: data.host,
+  };
+  createOneadSlot(global);
+  createAdUnit(global);
+}
+/**
+ * @param {!Window} win
+ */
+function createOneadSlot(win) {
+  const slot = document.createElement('div');
+  slot.id = 'onead-amp';
+  win.document.getElementById('c').appendChild(slot);
+}
+/**
+ * @param {!Window} win
+ */
+function createAdUnit(win) {
+  const src = 'https://ad-specs.guoshipartners.com/static/js/onead-amp.min.js';
+  const js = document.createElement('script');
```

Comment:
Oops! My apologies on missing this on the first review.

Could you use our helper function `loadScript` here?

https://github.com/ampproject/amphtml/blob/master/3p/3p.js#L100

And if you need to add any data to the script url, please be sure to use `encodeURIComponent`. E.g: https://github.com/ampproject/amphtml/blob/master/3p/github.js#L47